content/courses/advanced-ada/parts/data_types/records.rst

```diff
@@ -2918,20 +2928,684 @@ In this example, :ada:`Arr_Sub` and :ada:`Rec_Sub` are unconstrained subtypes.
    - :arm:`3.2 Types and Subtypes <3-2>`
 
 
-..
-    TO BE DONE:
+.. _Adv_Ada_Variant_Records:
+
+Variant parts
+-------------
+
+We've introduced variant records back in the
+:ref:`Introduction to Ada course <Intro_Ada_Variant_Records>`.
+In simple terms, a variant record is a record with discriminants that allows
+for changing its structure. Basically, it's a record containing a :ada:`case`
```

Comment:
Consider replacing "changing" with "varying" in order to tie it to the term "variant"...

Reply:
Good point!